3.Linear_Modeling/1_2.fit_linear_model_2b.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #55.            r2_score = lm.score(X=X, y=cp_subset_df)
</pre><p>Do you know the p-value too? Shouldn't be an issue if you have a lot of samples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>